source/installation/resource-manager/slurm.rst

```diff
@@ -33,6 +33,11 @@ adapter
   This is set to ``slurm``.
 cluster
   The Slurm cluster name. *Optional*, passed to SLURM as ``-M <cluster>``
+
+  .. warning::
+    Using the ``cluster`` option is discouraged. This is because maintenance
+    outages on the Slurm DB will propogate to Open OnDemand.  Instead sites
+    should use different ``conf`` files for each cluster to limit maintanence outages.
```

Comment:
@treydock IDK if this is exactly the right wording we want to use. Any suggestion would be appreciated.

Reply:
@treydock I'm going to merge this now, but could use your input as to whether this is the right wording we want to use or if indeed we want to leave the note at all.